scripts/extract-paired-reads.py

```diff
@@ -60,25 +87,45 @@ def main():
     info('extract-paired-reads.py')
     args = get_parser().parse_args()
 
-    check_input_files(args.infile, args.force)
-    infiles = [args.infile]
-    check_space(infiles, args.force)
-
-    outfile = os.path.basename(args.infile)
-    if len(sys.argv) > 2:
-        outfile = sys.argv[2]
-
-    single_fp = open(outfile + '.se', 'w')
-    paired_fp = open(outfile + '.pe', 'w')
-
-    print('reading file "%s"' % args.infile, file=sys.stderr)
-    print('outputting interleaved pairs to "%s.pe"' % outfile, file=sys.stderr)
-    print('outputting orphans to "%s.se"' % outfile, file=sys.stderr)
+    infile = args.infile
+    check_input_files(infile, args.force)
+    check_space([infile], args.force)
+
+    # decide where to put output files - specific directory? or just default?
+    if infile == '/dev/stdin':
```

Comment:
Are we not accepting `-` for the infile name (or otherwise defaulting to stdin?)